vendor/github.com/sylabs/sif/pkg/sif/fmt.go

```diff
@@ -89,6 +89,8 @@ func fstypeStr(ftype Fstype) string {
 		return "Archive"
 	case FsRaw:
 		return "Raw"
+	case FsEncrypt:
```

Comment:
This change requires a PR in https://github.com/sylabs/sif repository. We don't touch vendor